eQuiz/modules/eQuiz.Web/Areas/Admin/Scripts/layout.controller.js

```diff
@@ -0,0 +1,36 @@
+﻿(function (angular) {
+    angular.module('equizModule').controller('LayoutController', LayoutController);

```

Comment:
Use the following convention:

`angular
    .module('equizModule')
    .controller('LayoutController', LayoutController);`